examples/text-generation/run_generation.py

```diff
@@ -220,7 +226,11 @@ def setup_parser(parser):
         action="store_true",
         help="Preprocess on cpu, and some other optimizations. Useful to prevent recompilations when using dynamic prompts (simulate_dyn_prompt)",
     )
-
+    parser.add_argument(
+        "--assistant_model",
+        default=None,
+        help="Path to or name of the assistant model to use for assisted decoding.",
```

Comment:
where is assistant_model passed on to generate function?

Reply:
Addressed in run_sepculative_sampling to facilitate initialization of draft model which could not directly be done from present utils.